components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilder.java

```diff
@@ -252,29 +265,33 @@ private Integer fetchPort() {
 
         private String protocol;
         private String hostName;
```

Comment:
Shall we rename this for proxyHostName

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2949/commits/505c8f4e1f9f23a167ac4bac3b67bac5bab186cc